modules/fundamental/src/purl/endpoints/test.rs

```diff
@@ -310,3 +311,22 @@ async fn test_purl_license_details(ctx: &TrustifyContext) -> Result<(), anyhow::
     assert!(expected_result.contains_subset(response.clone()));
     Ok(())
 }
+
+#[test_context(TrustifyContext)]
+#[test(actix_web::test)]
+async fn garbage_collect(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+    setup(&ctx.db, &ctx.graph).await?;
+    let app = caller(ctx).await?;
+
+    let uri = "/api/v2/purl/gc";
+    let request = TestRequest::get().uri(uri).to_request();
+    let response = app.call_service(request).await;
+
+    assert_eq!(response.status(), StatusCode::OK);
+
+    let body_bytes = actix_web::test::read_body(response).await;
+    let body_str = std::str::from_utf8(&body_bytes)?;
+    assert_eq!(body_str, "8");
```

Comment:
This is currently affirming the bug that #1980 fixes, correct? Because I don't see anything being deleted that would generate any "garbage" to be collected.

I suppose we can merge this, and then rebase #1980 to change this to 0?

Reply:
https://github.com/guacsec/trustify/pull/1980 already affirms the fix :+1: 

> Because I don't see anything being deleted that would generate any "garbage" to be collected.

Yeah my focus was on the endpoint-call-only as the 1980 already is the actual test.

But you right, that is confusing. I'm reusing the `setup` function which ingests only packages with no parent sbom or advisory, so that they are orphans by default.

> I suppose we can merge this, and then rebase https://github.com/guacsec/trustify/pull/1980 to change this to 0?

Not exactly because we are not ingesting sbom or advisory in this test, just calling the endpoint :+1:

Reply:
Ah, I was wrongly assuming they weren't orphans. Since they are, the test makes sense.